### ISSUE TITLE
MISSING FILES: content.json

### ISSUE THREAD

**Issue by Shufflow80** [HelloZeroNet/ZeroNet#539]
Hi,
I created a site on zeronet but when I go in the sidebar there is written MISSING FILES :
content.json
and when I try to connect from another computer there is written no peer found, 
I created a site on zeronet but when I go in the sidebar there is written : NO PEERS FOUND
Can you help me?

**Comment by OliverCole** [comment]
How did you create the site? Did you sign and publish it?

**Comment by Shufflow80** [comment]
I followed this tutorial zeronet.readthedocs.io/en/latest/using_zeronet/create_new_site/

**Comment by HelloZeroNet** [comment]
It's normal, you can ignore it. To make it accessible from other computer you need an open port or tor installed.

**Comment by dotnetspec** [comment]
I have the same problem. I published a site on my laptop which I was then able to connect to via my Android phone via Chrome. I've updated the site but now the Android Chrome states 'no peers found'. I am unable to open port 15441 in Android as, to do this, I was advised to:
Tap "Settings | Wireless & Networks | Wi-Fi." Touch the "Menu" button and then press "Advanced." Write down the information to the right of the IP Address field. My phone reports 'IP address unavailable'. 

I then installed Orfox: Tor on the Android but it reports 'Unable to connect' when I try to access the site via Tor (so I got further with Chrome).

My laptop reports that port 11460 is closed but 'your tor gateway is running well'. I opened port 11460 previously but it made no difference to the 'port closed' label. Is there anything else I can do to enable my Android to find a peer (my laptop) and update the site?
Should this issue be re-opened?
